var/spack/repos/builtin/packages/vacuumms/package.py

```diff
@@ -20,6 +22,19 @@ class Vacuumms(CMakePackage):
     maintainers = ['frankwillmore']
 
     version('master', branch='master')
-    version('1.0.0', 'c18fe52f5041880da7f50d3808d37afb3e9c936a56f80f67838d045bf7af372f')
-
-    depends_on('libtiff')
+    version('1.1.1', branch='v1.1.1')
```

Comment:
I'm not seeing this as a branch.  It would be clearer if you used the tag:

```suggestion
    version('1.1.1', tag='v1.1.1')
```